### ISSUE TITLE
TypeError: Vue.component is not a function

### ISSUE THREAD

**Issue by wuxianliang** [kazupon/vue-server-express-demo-example#1]
Dear @kazupon :
Thank you for this demo, it helps me to understand vue-server. But when I install this demo,  this error prevent me from starting. I have to remove this line to start the demo.

```
Vue.component('item', require('./components/item'))
    ^

TypeError: Vue.component is not a function
    at Object.<anonymous> (/Users/wxl/asdf/vue-server-express-demo-example/routes.js:5:5)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/wxl/asdf/vue-server-express-demo-example/app.js:8:16)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)

```

**Comment by kazupon** [comment]
Thank you for your reporting !!

I think that this issue is bug of vue-server.
Today, `vue-server` have just  published new version (0.4.0-beta.6).
I pinned `vue-server` version to 0.4.0-beta.5 just now.

Please try it again. :smile_cat:

**Comment by wuxianliang** [comment]
Thank you very much. It is fixed.
